test/test_datapipe.py

```diff
@@ -48,6 +52,26 @@
 )
 
 
+def test_torchdata_pytorch_consistency():
```

Comment:
I just went over the PR again. Correct me if I am wrong, this method is actually not being carried out during testing. Could you please wrap it into a new TestCase?

Reply:
If you run this with `pytest` it will be run, since `pytest` doesn't require test classes. Are you using `unittest`as a runner internally?

Reply:
Oh, thanks for letting me know. Then it's fine, I am using `pytest`. I should change CI to run pytest in verbose mode